src/frontend/screens/JoinProject.tsx

```diff
@@ -0,0 +1,125 @@
+import WifiBar from "../sharedComponents/WifiBar";
+
+const m = defineMessages({
+  title: {
+    id: "screens.JoinProject.title",
+    defaultMessage: "Join a Project",
+  },
+  instructionsTitle: {
+    id: "screens.JoinProject.instructionsTitle",
+    defaultMessage: "Instructions",
+  },
+  instructionsDescription: {
+    id: "screens.JoinProject.instructionsDescription",
+    defaultMessage: "Show this QR code to your Project Admin",
+  },
+  sendJoinRequest: {
+    id: "screens.JoinProject.sendJoinRequest",
+    defaultMessage: "Send Join Request instead",
+  },
+});
+
+const JoinProject = () => {
+  const { ssid } = useWifiStatus();
+
+  const deviceName: string =
+    "Android " + getUniqueId().slice(0, 4).toUpperCase();
+
+  return (
+    <View style={styles.pageContainer}>
+      {/* TODO: Probably want to do something smarter here if no ssid is detected */}
+      {ssid && <WifiBar deviceName={deviceName} ssid={ssid} />}
+      <View style={styles.container}>
+        <View>
+          <View style={styles.qrCodeContainer}>
+            <QRCode value="https://digital-democracy.org" size={250} />
+          </View>
+          <View style={styles.instructionsContainer}>
+            <Text style={styles.instructionsTitle}>
+              <FormattedMessage {...m.instructionsTitle} />
+            </Text>
+            <Text style={styles.instructionsDescription}>
+              <FormattedMessage {...m.instructionsDescription} />
+            </Text>
+          </View>
+        </View>
+        <Button
+          variant="text"
+          onPress={() =>
+            Alert.alert("Work in progress", "Feature not implemented yet")
+          }
+        >
+          <Text style={styles.sendJoinRequest}>
+            <FormattedMessage {...m.sendJoinRequest} />
+          </Text>
+        </Button>
+      </View>
+    </View>
+  );
+};
+
+JoinProject.navigationOptions = () => ({
+  headerTitle: () => (
+    <HeaderTitle style={{ color: WHITE }}>
+      <FormattedMessage {...m.title} />
+    </HeaderTitle>
+  ),
+  headerLeft: ({ onPress }: { onPress: () => void }) => (
+    <IconButton onPress={onPress}>
+      <BackIcon color={WHITE} />
+    </IconButton>
+  ),
+  headerStyle: {
+    backgroundColor: MEDIUM_BLUE,
+  },
+});
+
+const styles = StyleSheet.create({
+  pageContainer: {
+    flex: 1,
+  },
+  container: {
+    alignItems: "center",
+    justifyContent: "space-between",
+    flex: 1,
+    padding: 40,
+  },
+  qrCodeContainer: {
+    alignItems: "center",
+  },
+  instructionsContainer: {
+    alignItems: "center",
+    marginVertical: 40,
+  },
+  instructionsTitle: {
+    fontSize: 24,
+    fontWeight: "bold",
+    marginBottom: 10,
+  },
+  instructionsDescription: {
+    fontSize: 16,
+  },
+  joinRequestContainer: {
+    alignItems: "center",
+  },
+  sendJoinRequest: {
+    fontSize: 16,
+    fontWeight: "bold",
+    textDecorationLine: "underline",
+  },
+});
+
+export default JoinProject;
```

Comment:
very open to renaming this component. just needed something that was clear enough to work with as i was building